atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -151,7 +151,13 @@ def adjust_for_autorebuild(self):
     def adjust_for_inherit(self):
         if not self.plugin_result:
             return
-        build_info = self.plugin_result[BASE_IMAGE_KOJI_BUILD]
+
+        build_info = self.plugin_result.get(BASE_IMAGE_KOJI_BUILD)
+        if build_info is None:
+            self.log.debug('No base image koji build info is found. '
+                           'Skip to adjust inheritance for composes.')
```

Comment:
@tkdchen; Based on this comment, I'm currently confused as to what's being skipped and why. Do you perhaps mean "Skipping forward to adjust inheritance for composes."?

Reply:
@ben-alkov Yeah, you are right. Sorry, "skip to" was used incorrectly.